src/friction-compensation.cpp

```diff
@@ -0,0 +1,40 @@
+#include "simple-mpc/friction-compensation.hpp"
+
+namespace simple_mpc
+{
+
+  FrictionCompensation::FrictionCompensation(const Model & model, const bool with_free_flyer)
+  {
+    if (with_free_flyer)
+    {
+      // Ignore universe and root joints
+      nu_ = model.njoints - 2;
+    }
+    else
+    {
+      // Ignore root joint
+      nu_ = model.njoints - 1;
```

Comment:
Shouldn't it be `nu_ = model.nv - 6` ?

Reply:
Same comment: both are equivalent (in this case, nu = model.nv because no free flyer)

Reply:
This is unfortunately not true. Some joints have more than 1 dof.. (or less than 1 sometimes ...)

Reply:
I guess if we want to cover all the edge cases, you're right...